semgrep/semgrep/output_from_core.py

```diff
@@ -283,7 +283,7 @@ def to_json_string(self, **kw: Any) -> str:
         return json.dumps(self.to_json(), **kw)
 
 
-@dataclass
+@dataclass(frozen=True)
```

Comment:
So do we want to commit the generated code in the repository? Simpler for people?

Reply:
I tried removing it but I got pre-commit linter errors, due to an attempt to resolve names. Not getting this error would mean running atdpy before running the pre-commit linters, which seemed too complicated.